src/lib/getClasses.ts

```diff
@@ -6,6 +6,16 @@ import {
   IRenderer,
 } from '../../types/theme'
 
+import * as crypto from 'crypto'
+
+const getHashCode = (fromString: string): string => {
+  return crypto
+    .createHash('md5')
```

Comment:
if you are about the part where just first five characters are taken from calculated hash - have done this only for this prototype - but there are plenty of other strategies we might consider to make this 'id' part readable. 

But if you are talking about hash value in general - yes, agree, it will be extremely 'funny', but good point is that probability of that is low in a reasonable proportion to this 'funny' part :)